content/blog/monthly-july-19.md

```diff
@@ -0,0 +1,133 @@
+---
+title: This Month in TiKV: July 2019
+date: 2019-08-07
+---
+
+Hi! Welcome to the first ever edition of 'This Month in TiKV', covering July 2019. As the name suggests, this is a monthly newsletter covering interesting things happening in the world of [TiKV](https://tikv.org/), an open-source, distributed key-value store.
+
+We're just getting started with the newsletter, and you should expect it to evolve as we go along. We hope to cover news and events of interest to TiKV contributors and users, significant PRs and issues in the TiKV [repo](https://github.com/tikv/tikv), and generally keep you informed of what is going on. Since this is the first issue we'll cover a few bits of news from before July.
+
+You'll find the newsletter [on the TiKV blog](https://tikv.org/blog/) from the first week of each month. We should have a mailing list so you can receive the newsletter by email soon.
+
+[Let us know](https://github.com/nrc/tmit/issues/new) what you think!
```

Comment:
```suggestion
[Let us know](https://github.com/tikv/website/issues/new) what you think!
```

We shouldn't be hosting this on a personal repo. Maybe better to point them at the website?